docs/cookbook.rst

```diff
@@ -50,15 +57,93 @@ adding a ``crispy`` template filter to the form. For example:
     <form action="{% url 'envelope-contact' %}" method="post">
         {% csrf_token %}
         {% antispam_fields %}
-        {{ form|crispy }}
-        <button type="submit">Send!</button>
+        {{ crispy form }}
     </form>
 
 .. _`Bootstrap`: http://getbootstrap.com/
 .. _`django-crispy-forms`: https://github.com/maraujop/django-crispy-forms
 
+To add a submit button, create a custom form using ``django-crispy-forms`` helper:
+
+# forms.py
+    from envelope.forms import ContactForm
+    from crispy_forms.helper import FormHelper
+    from crispy_forms.layout import Submit
+
+
+    class MyContactForm(ContactForm):
+        def __init__(self, *args, **kwargs):
+            super(MyContactForm, self).__init__(*args, **kwargs)
+            self.helper = FormHelper()
+            self.helper.add_input(Submit('submit', 'Submit', css_class='btn-lg'))
+
+And finally link this form to your view:
+
+# views.py
+    from braces.views import FormMessagesMixin
+    from envelope.views import ContactView
+
+    from django.utils.translation import ugettext_lazy as _
+
+    from .forms import MyContactForm
+
+
+    class MyContactView(FormMessagesMixin, ContactView):
+        form_invalid_message = _(u"There was en error in the contact form.")
+        form_valid_message = _(u"Thank you for your message.")
+        form_class = MyContactForm
+
+or just use it in your urls.py if you directly reference :method:``envelope.views.ContactView.as_view`` :
+
+# urls.py
+    from django.conf.urls import patterns, url
+    from envelope.views import ContactView
+
+    from .forms import MyContactForm
+
+
+    urlpatterns = patterns('',
+        url(r'^contact/', ContactView.as_view(form_class=MyContactForm)),
+    )
+
+Displaying form messages nicely
+-------------------------------
+
+GETting the contact form page after POSTing it will give you access to either a success message (form_valid_message)
+or an error message (form_invalid_message) thanks to django-braces' :class:``FormMessagesMixin``. These messages use
```

Comment:
The class name will not be a usable link unfortunately, as it's external to the project. Either change it to a regular Markdown link, or perhaps use `intersphinx` for cross-project documentation (I'm not sure if/how it works with readthedocs, but might be worth looking into). Currently the Django docs are linked with intersphinx (see `conf.py` for example).